recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.13.1" %}
+{% set version = "0.14.0" %}
```

Comment:
I have a question about that regarding geos and features. See https://github.com/conda-forge/geos-feedstock/pull/5#issuecomment-201940475

Not sure how that will affect cartopy.